consensus/src/liveness/leader_reputation_test.rs

```diff
@@ -199,28 +203,29 @@ impl Example1 {
 #[test]
 fn test_aggregation_counting() {
     let mut example1 = Example1::new();
-    let validators = example1.validators.clone();
+    let validators = example1.validators0.clone();
+    let epoch_to_validators = HashMap::from([(0u64, validators.clone())]);
```

Comment:
Can you add some test scenarios where we consider the history of multiple epochs with some overlapping validators and few non-overlapping ones?